src/pages/forget-password.tsx

```diff
@@ -11,22 +11,16 @@ import { useForgetPasswordMutation } from '@generated/graphql.queries';
 export default function ForgetPassword() {
   const { t } = useTranslation('auth');
   const head = { title: t('forget-password.title') };
-
-  const [successMessage, setSuccessMessage] = useState('');
   const { gqlClient } = useApiClient();
   const {
     mutate: forgetPasswordSendEmail,
     error,
     isLoading: isSendingEmail,
   } = useForgetPasswordMutation(gqlClient, {
-    onSettled: (data, error) => {
+    onSettled: (error) => {
       if (error) {
         // @ts-ignore
         console.error(error);
-        setSuccessMessage('');
-      }
-      if (data) {
-        setSuccessMessage(t('forget-password.success'));
```

Comment:
Here the `error` and `data` are two situations when the `useForgetPasswordMutation` is called,  `error` means the request failed, so we log out the error to see the error, and `data` means we successfully received the data back from the backend, so the `router.push(AuthRoute.forgetPasswordSuccess)` should go in the `if(data)` condition.